samples/ipc_call_interface/src/math.c

```diff
@@ -31,13 +31,13 @@ static az_ulib_result sum_concrete(const void* const model_in, const void* model
   return AZ_ULIB_SUCCESS;
 }
 
-static az_ulib_result subtraction_concrete(const void* const model_in, const void* model_out) {
+static az_ulib_result sub_concrete(az_ulib_model_in model_in, az_ulib_model_out model_out) {
```

Comment:
nit:
```suggestion
static az_ulib_result subtract_concrete(az_ulib_model_in model_in, az_ulib_model_out model_out) {
```
only because sub can also be interpreted as substitute.

Reply:
Now I remember that we had this discussion. I'll rollback the name to subtract. 

---
In reply to: [529744829](https://github.com/Azure/azure-ulib-c/pull/53#discussion_r529744829) [](ancestors = 529744829)